kura/distrib/config/kura.build.properties

```diff
@@ -34,6 +34,12 @@ org.eclipse.kura.linux.net.version=1.0.11-SNAPSHOT
 org.eclipse.kura.linux.net.test.version=1.0.4-SNAPSHOT
 org.eclipse.kura.linux.position.version=1.0.7-SNAPSHOT
 org.eclipse.kura.linux.usb.version=1.0.8-SNAPSHOT
+org.eclipse.kura.linux.usb.armv5sf.version=1.0.0-SNAPSHOT
```

Comment:
Hi Jens,
why the fragments, in this case, have a version that is different of the  host one? This is different if compared to jdk.dio.